From 51e7621a6bd4b0b7a6d741d56bac925c665f5b9b Mon Sep 17 00:00:00 2001
From: Catalin Popescu <catalin.popescu@intel.com>
Date: Wed, 13 Apr 2016 17:40:45 +0200
Subject: [PATCH 33/41] Dollar Cove: add GPIO1 regulator.

Allow GPIO1 regulator to be used by the drivers.

Change-Id: I191b4e3a2aa6a6aa884e2333205af04cb456e483
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-31683
Signed-off-by: Catalin Popescu <catalin.popescu@intel.com>
Reviewed-on: https://android.intel.com:443/494298
Reviewed-on: https://android.intel.com:443/499890
---
 drivers/mfd/intel_soc_pmic_dc.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/mfd/intel_soc_pmic_dc.c b/drivers/mfd/intel_soc_pmic_dc.c
index 25839e9..2046fe2 100644
--- a/drivers/mfd/intel_soc_pmic_dc.c
+++ b/drivers/mfd/intel_soc_pmic_dc.c
@@ -273,6 +273,12 @@ static struct mfd_cell dollar_cove_dev[] = {
 		.num_resources = 0,
 		.resources = NULL,
 	},
+	{
+		.name = "dcovex_regulator",
+		.id = DCOVEX_ID_GPIO1 + 1,
+		.num_resources = 0,
+		.resources = NULL,
+	},
 	{NULL, },
 };
 
-- 
1.9.1

